cmd/juno/juno.go

```diff
@@ -73,6 +74,7 @@ const (
 	defaultMetricsPort         = 9090
 	defaultGRPC                = false
 	defaultGRPCPort            = 6064
+	defaultRPCMaxBlockScan     = 64
```

Comment:
64 blocks seems a bit too few to me, did you run any tests to see how long 64 blocks take to scan?

Reply:
It takes around 10ms on my machine when each block is early skipped due to bloomfilter.

Reply:
drive-by comment: if I'm understanding correctly, this is a significant change to the default behavior. Can we set the default to the max uint64 to minimize breaking changes? Users can explicitly set the limit if they want. I believe geth doesn't set a limit by default, but feel free to fact check me.

Reply:
geth doesn't have such option at all. There is an open [issue](https://github.com/ethereum/go-ethereum/issues/24508#issuecomment-1063843495) for that

Reply:
> Can we set the default to the max uint64 to minimize breaking changes?

This is not really a breaking change IMO. Users should already be able to handle pagination.

Reply:
>  It takes around 10ms on my machine when each block is early skipped due to bloomfilter.

Then, we can bump it to 512 IMO.